### ISSUE TITLE
learning rate

### ISSUE THREAD

**Issue by remybonnav** [Linaqruf/kohya-trainer#81]
Hi, why are there 3 values for learning raate
learning rate
unet lr 
textencoder lr

why not only 2 (unet lr and text encoder lr).elected 
For example I just run a training with only unet learning, learning rate 1.E-4 and unet lr = 0 (I did not see that there was many settigns for learning rate).
In this case, is my learning rate 1.E-4 killed because I left 0 in unet lr?

**Comment by Linaqruf** [comment]
it used to only unet and tenc lr, but i put learning rate back by request

so if you set tenc to 5e-5, then and you set global lr to 1e-4, so unet = global_lr

**Comment by remybonnav** [comment]
Ok, in the meantime I saw on the kohya repo that if nothing is written in
unet lr then it takes into account the global lr (but I was not sure if "0"
in unet lr would be considered as a value or as nothing)

**Comment by Linaqruf** [comment]
look at train_util.py, there is learning_rate there

**Comment by Linaqruf** [comment]
I'm closing this issue, let me know if you facing the error!
